.env.production.json

```diff
@@ -50,5 +50,558 @@
+  "contractBlockNo": 13553873,
+  "contractABI": [
```

Comment:
why don't we use the human readable contract interface?

```js
> var contract = new ethers.Contract(env.contractAddress, env.contractABI, provider);
undefined
> contract.interface.format()
[
  'constructor(string name_, string symbol_, uint16 supply_, string sharedBaseURI_)',
  'event Approval(address indexed owner, address indexed approved, uint256 indexed tokenId)',
  'event ApprovalForAll(address indexed owner, address indexed operator, bool approved)',
  'event LogbookNewLog(uint256 tokenId, uint256 index, address sender)',
  'event LotteryWinners(address[] winners)',
  'event OwnershipTransferred(address indexed previousOwner, address indexed newOwner)',
  'event PreOrderMinted(address sender, uint256 n)',
  'event Transfer(address indexed from, address indexed to, uint256 indexed tokenId)',
  'function _randomDraw(address[] addresses_, uint256 amount_) view returns (address[] result)',
  'function appendLog(uint256 tokenId_, string message_)',
  'function approve(address to, uint256 tokenId)',
  'function balanceOf(address owner) view returns (uint256)',
  'function batchMint(address[] addresses_, uint16 amount_) returns (uint256[])',
  'function burn(uint256 tokenId_)',
  'function contractURI() view returns (string)',
  'function drawLottery(address[] addresses_, uint256 amount_)',
  'function getApproved(uint256 tokenId) view returns (address)',
  'function inPreOrder() view returns (bool)',
  'function isApprovedForAll(address owner, address operator) view returns (bool)',
  // ...

> var contract = new ethers.Contract(env.contractAddress, ABI, provider);
undefined
> contract.interface.format()          // same as above

```

Reply:
It would be great though need a format conversion!